data/common.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 duo_authproxy::version: 2.7.0
+duo_authproxy::use_systemd: true
```

Comment:
This changes behavior for a non major version. Maybe this should be false by default at first?